train/experience.py

```diff
@@ -1,16 +1,25 @@
+"""
+Experience class definition
+"""
 import numpy as np
 
 from train.reward import allRewards, rawRewards
 
 
 class Experience:
+    """
+    Experience class to store moves, rewards and metric values
+    """
     def __init__(self):
         self.moves = np.array([])
         self.rewards = np.array([])
 
         self.metric = np.array([])
 
     def add_episode(self, game_states, moves):
+        # moves is not used here, kept for inheritance reasons
+        # TODO Edouard to act on this
+        # pylint: disable=W0612,W0613
```

Comment:
@Edouard360 : The argument `moves` here is not used by the method.
I understand that it is use by the Inherited class `ExperienceVanilla` but I still believe this is not good practice. Haven't had time to think of a solution though :)